pygments/lexers/gsql.py

```diff
@@ -0,0 +1,78 @@
+import re
+
+from pygments.lexer import RegexLexer, include, bygroups, using, this
+from pygments.token import Keyword, Punctuation, Comment, Operator, Name,\
+    String, Number, Whitespace
+
+
+__all__ = ["GSQLLexer"]
+
+class GSQLLexer(RegexLexer):
```

Comment:
Please add a comment describing this lexer -- ideally with a link to the language definition. This comment should end with `.. versionadded:: 2.10`

Reply:
@DanBarkus thanks for the great work :smile: 
@Anteru I can't wait to see this PR Merged :smile:

Reply:
@DanBarkus Very cool! Can't wait to use it!
@Anteru Looking forward to the merge!